Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -154,7 +148,7 @@ public virtual void Start()
             // some transports might not be ready until Start.
             //
             // (tick rate is applied in StartServer!)
-            if (IsHeadless && startOnHeadless)
+            if (NetworkServer.IsHeadless && startOnHeadless)
```

Comment:
maybe we don't need the method at all?  how about:
```suggestion
            if (SystemInfo.graphicsDeviceType == GraphicsDeviceType.Null && startOnHeadless)
```

Reply:
I am ok with removing it.